frontend/packages/pipelines-plugin/src/components/shared/workspaces/WorkspaceResourceLinkList.tsx

```diff
@@ -29,72 +29,70 @@ const WorkspaceResourceLinkList: React.FC<WorkspaceResourceLinkListProps> = ({
-                key={workspace.name}
-                name={configMap.name}
-                namespace={namespace}
-                kind={ConfigMapModel.kind}
-                displayName={displayName}
-              />
-            );
-          }
-          if (workspace.secret) {
-            const secret = workspace.secret as VolumeTypeSecret;
-            const displayName = `${secret.secretName} (${workspace.name})`;
-            return (
-              <ResourceLink
-                key={workspace.name}
-                name={secret.secretName}
-                kind={SecretModel.kind}
-                namespace={namespace}
-                displayName={displayName}
-              />
-            );
-          }
-          if (!workspace.volumeClaimTemplate) {
-            return <div key={workspace.name}>{workspace.name}</div>;
-          }
-          return null;
-        })}
-        <VolumeClaimTemplatesLink
-          namespace={namespace}
-          ownerResourceName={ownerResourceName}
-          ownerResourceKind={ownerResourceKind}
-        />
-      </dd>
-    </dl>
+    <>
+      <dl>
+        <dt>{t('pipelines-plugin~Workspace Resources')}</dt>
+        <dd>
+          {workspaces.map((workspace) => {
+            if (workspace.persistentVolumeClaim) {
+              const persistentVolumeClaim = workspace.persistentVolumeClaim as VolumeTypePVC;
+              const displayName = `${persistentVolumeClaim.claimName} (${workspace.name})`;
+              return (
+                <ResourceLink
+                  key={workspace.name}
+                  name={persistentVolumeClaim.claimName}
+                  namespace={namespace}
+                  kind={PersistentVolumeClaimModel.kind}
+                  displayName={displayName}
+                />
+              );
+            }
+            if (workspace.configMap) {
+              const configMap = workspace.configMap as VolumeTypeConfigMaps;
+              const displayName = `${configMap.name} (${workspace.name})`;
+              return (
+                <ResourceLink
+                  key={workspace.name}
+                  name={configMap.name}
+                  namespace={namespace}
+                  kind={ConfigMapModel.kind}
+                  displayName={displayName}
+                />
+              );
+            }
+            if (workspace.secret) {
+              const secret = workspace.secret as VolumeTypeSecret;
+              const displayName = `${secret.secretName} (${workspace.name})`;
+              return (
+                <ResourceLink
+                  key={workspace.name}
+                  name={secret.secretName}
+                  kind={SecretModel.kind}
+                  namespace={namespace}
+                  displayName={displayName}
+                />
+              );
+            }
+            if (!workspace.volumeClaimTemplate) {
+              return <div key={workspace.name}>{workspace.name}</div>;
+            }
```

Comment:
What scenario is this section for?

Reply:
This is listing the `emptyDir`s

Reply:
Interesting... not sure I like the if condition or how it renders... personally I think it should be removed if you are not handling the case, but I'll fix it in a follow up PR.

Thanks for the info.